libs/src/sanityChecks/index.js

```diff
@@ -15,6 +17,8 @@ class SanityChecks {
    */
   async run (creatorNodeWhitelist = null) {
     await isCreator(this.libs)
+    await sanitizeNodes(this.libs)
+    await addSecondaries(this.libs)
```

Comment:
not sure if this matters but prob we should sanitize after adding secondaries intead of before

Reply:
If the secondaries are not sanitized and we try to add more secondaries, we're gonna have problems. We could sanitize *again* after adding secondaries and potentially after rolling over, but I'd like to confirm that that's still a possibility before we add more checks

Reply:
makes sense